src/sqlfluff/core/templaters/slicers/tracer.py

```diff
@@ -89,10 +97,13 @@ def trace(self) -> JinjaTrace:
                 value = [m_id.group(0), p[len(m_id.group(0)) + 1 :], False]
```

Comment:
Just above this are two more UUID references in comments (I think). Should they be updated.

Sorry can't comment on unchanged lines.